controller_interface/CMakeLists.txt

```diff
@@ -13,16 +13,19 @@ endif()
 find_package(ament_cmake REQUIRED)
 find_package(hardware_interface REQUIRED)
 find_package(rclcpp_lifecycle REQUIRED)
+find_package(yaml_cpp_vendor REQUIRED)
 
 add_library(controller_interface SHARED src/controller_interface.cpp)
 target_include_directories(
   controller_interface
   PRIVATE
-  include)
+  include
+  "yaml_cpp_vendor")
```

Comment:
this line is superfluous when also being specified as `ament_target_dependencies` as seen below.

Reply:
Fixed in [1256483](https://github.com/ros-controls/ros2_control/pull/64/commits/1256483a2c13a9307042332b31eed767cc244ac8)